### ISSUE TITLE
Some CLI questions

### ISSUE THREAD

**Issue by BobSelf** [thetatoken/guardian-mainnet-guide#26]
I have some questions about running theta (cli) on Ubuntu linux 20.04 server. I've searched for documentation but couldn't find answers. I installed it with the script in the docs.
I started the node with "theta start --config=../guardian_mainnet/node" and things started happening.
Then I ran "thetacli query status" from another terminal and got
"Failed to get account details: Post http://localhost:16888/rpc: dial tcp 127.0.0.1:16888: connect: connection refused"
But after about 15 minutes I connected and got:
" {
	"address": "i...",
	"chain_id": "mainnet",
	"current_epoch":
	"current_height":
	"current_time":
	"latest_finalized_block_epoch":
	"latest_finalized_block_hash":
	"latest_finalized_block_height":
	"latest_finalized_block_time":
	"peer_id": "...",
	"syncing": false
}"

So, it looks like things were working fine. That was late yesterday. I control-c'd out of it and got back to it today. Now it's been running for an hour and the latest latest_finalized_block_time is about 8 hours ago. It's progressing, but very slowly and synching is true. What did I do wrong? Why is syncing so slow today but really fast yesterday?

1. I see that "theta start .... " is a way to start the node. But what is the proper way to stop it? I may want to reboot or just stop it for a few minutes.
2. Then how do you restart theta? "Screen theta --config=../guardian_mainnet/node"? Shouldn't I set this up as a service?
3. If I stake my theta to this node and this computer crashes are my tokens lost to me? Can I unstake them if this node no longer exists? Will the password simply rebuild the same node later?
4. Is there some documentation about these questions? Or a support forum?

**Comment by jieyilong** [comment]
Yeah, the RPC server thread starts only after the snapshot is loaded. So need to wait a bit after starting the `theta` process. The syncing speed depends on which peers your node is connected to, so it is expected that the syncing speed varies. 

1. To stop the `theta` process, you can simply do "ctrl+C".

2. And yes, you can run the `theta process` as a background process, just run it with `theta start --config=<CFG_FOLDER> --password=<PASSSWORD> &`.

3. No, you won't lose your token if the computer crashes. You can unstake from your wallet anytime, even after the node is stopped. To transfer the node to another machine, you'd need to copy over the `guardian_mainnet/node/key` folder, and use the same password to unlock it on the new maches.

4. For more details, please refer to the Theta documentation site: https://docs.thetatoken.org/

Hope this helps!
